src/AWS.Deploy.Recipes/RecipeDefinitions/aws-deploy-recipe-schema.json

```diff
@@ -352,7 +352,6 @@
                         "type": "string",
                         "enum": [
                             "FargateTaskSizeCpuMemoryLimits",
-                            "MinMaxConstraint",
```

Comment:
I can't comment on it directly, but could you also remove the related block starting at line 383?

And since "Comparison" has properties, can you add the property typing block to the schema?

Reply:
Done